README.md

```diff
@@ -77,7 +77,8 @@ showLocation({
     dialogTitle: 'This is the dialog Title', // optional (default: 'Open in Maps')
     dialogMessage: 'This is the amazing dialog Message', // optional (default: 'What app would you like to use?')
     cancelText: 'This is the cancel button text', // optional (default: 'Cancel')
-    appsWhiteList: ['google-maps'] // optionally you can set which apps to show (default: will show all supported apps installed on device)
+    appsWhiteList: ['google-maps'], // optionally you can set which apps to show (default: will show all supported apps installed on device)
+    appTitles: { 'google-maps': 'My custom Google Maps title' } // optionally you can override default app titles
```

Comment:
Should we maybe leave this an empty object by default, like we do in the popup example? Just to make sure that people don't simply copy and paste this into their code and have a weird 'My custom Google Maps title' pop up.

Reply:
Oh! Or commenting it out, like in the line below it. That might be better.

Reply:
OK, I understand. I have commented this line.